packages/field-content/src/Block.js

```diff
@@ -1,4 +1,4 @@
-export class Block {
+export default class Block {
```

Comment:
I believe we made a decision a bit ago to not use default exports in package entrypoints. I noticed default exports in some other places too, why don't we have a single entrypoint with named exports?

Reply:
Oh yeah, strongly agree on this ☝️

Reply:
Oh yeah, I'll fix that up. Good catch! 👍

Reply:
Fixed.